### ISSUE TITLE
Smart Enroll - Problems with the recognition of CC

### ISSUE THREAD

**Issue by NicolasHT95** [Open-Verifik/web-sdk#38]
## DESCRIPTION

I was helping sales team to create some projects for specific client, the idea is that they make a Demo showing the services of Verifik an how they work, in that process we found this unusual situation using the Colombian ID of Manuel.

## TO-DO

Check the following video, we found this problem using the Smart Check OCR solution

https://verifikco-my.sharepoint.com/:v:/g/personal/nicolas_verifik_co/EXwjFgrHqCpAnSlesLSiizoB0GcYe7BW5eNh333FisZ8NA?e=7MDtZi&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D

Basically with the document of Manuel the service don't work, it don't proccess the image but using the scan prompt service works just fine.

![Image](https://github.com/Open-Verifik/web-sdk/assets/138807516/11d57a3d-663f-4f10-b176-dee3a651b942)

here are the images used in the test

![Image](https://github.com/Open-Verifik/web-sdk/assets/138807516/a30a706a-f965-4079-bce0-9045a445c170)
![Image](https://github.com/Open-Verifik/web-sdk/assets/138807516/2fbb9b8c-2d3f-48af-88c8-fb2d5ee5d03b)

**Comment by migueltrevinom** [comment]
@NicolasHT95 check it now please

**Comment by NicolasHT95** [comment]
**QA was successful**
